<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportPasswordWidget</class>
 <widget class="QWidget" name="ExportPasswordWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>80</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <item row="3" column="0">
    <spacer name="passwordSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2" alignment="Qt::AlignTop">
    <widget class="QCheckBox" name="cryptCheckBox">
     <property name="text">
      <string>Protect with password</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="nx::vms::client::desktop::PasswordInputField" name="passwordEdit" native="true">
     <property name="placeholderText" stdset="0">
      <string>Password</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::PasswordInputField</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/password_input_field.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
